lib/elixir/lib/map.ex

```diff
@@ -1149,6 +1149,44 @@ defmodule Map do
     end
   end
 
+  @doc """
+  Intersects two maps into one.
+
+  If a key exists in both maps, the value in `map2` will be used.
```

Comment:
Nitpick: the key has to exist in both maps in order to be in the returned map. So I think we can make this sentence a little bit clearer:

```suggestion
  The values in the returned map are the values of the intersected keys in `map2`.
```